arcgis-ios-sdk-samples/Augmented reality/View hidden infrastructure in AR/README.md

```diff
@@ -0,0 +1,64 @@
+# View hidden infrastructure in AR
+
+Visualize hidden infrastructure in its real-world location using augmented reality.
+
+![Add pipe infrastructure to the map](view-hidden-infrastructure-in-AR-1.png)
+![View hidden infrastructure in AR](view-hidden-infrastructure-in-AR-2.png)
+
+## Use case
+
+You can use AR to "x-ray" the ground to see pipes, wiring, or other infrastructure that isn't otherwise visible. You could use this feature to trace the flow of water through a building to help identify the source of a leak.
+
+## How to use the sample
+
+When you open the sample, you'll see a map centered on your current location. Use the sketch editor to draw pipes around your location. An elevation surface will be queried to place the drawn infrastructure above or below ground. When you're ready, tap the camera button to view the infrastructure you drew in AR.
+
+There are two calibration modes in the sample: roaming and local. In roaming calibration mode, your position is updated automatically from the location data source every second. Because of that, you can only adjust heading, not position or elevation. This mode is best when working in a large area, where you would travel beyond the useful range of ARKit.
+
+When you're ready to take a more precise look at the infrastructure, switch to local calibration mode. In local calibration mode, you can make fine adjustments to location, altitude, and heading to ensure the content is exactly where it should be.
+
+## How it works
+
+1. Draw pipes on the map. See more in the "Sketch on map" sample to learn how to use the sketch editor for creating graphics.
```

Comment:
It is possible to do, but from previous experience we typically don't do it, probably because the links might expire when repo structure/developer site urls change.

Reply:
I see. I think there was a readme that linked to a different sample in its "Additional information" section but I don't remember which sample it was.